priv/stats_descriptions.cfg

```diff
@@ -52,8 +52,14 @@
     {desc, <<"length of an dreyfus_index info request">>}
 ]}.
 
+%% Declare IOQ search channel metrics
+{[couchdb, io_queue, search], [
+    {type, counter},
+    {desc, <<"Search IO directly triggered by client requests">>}
+]}.
+
```

Comment:
Good catch, I forgot to add the associated metric for IOQ1. This metric is updated by https://github.com/apache/couchdb-ioq/blob/master/src/ioq_server.erl#L424 and we can see it's of the format `[couchdb, io_queue, search]` and this change is correct.